pkg/obfuscator/multi_obfuscator.go

```diff
@@ -0,0 +1,46 @@
+package obfuscator
+
+type MultiObfuscator struct {
+	obfuscators []ReportingObfuscator
+}
+
+func (m *MultiObfuscator) Path(s string) string {
```

Comment:
What is the advantage here to return a single string after the multiple obfuscations?

Reply:
this is the business logic as described in the design doc: https://docs.google.com/document/d/1IDUfaw2IOPo7fRK142rsiAcxwl7tAHXJwLhaoyhambI/edit#

you can also read more about the behaviour in the readme: https://github.com/openshift/must-gather-clean#chaining-obfuscators-and-side-effects